appveyor.yml

```diff
@@ -0,0 +1,24 @@
+environment:
+  RUST_MIN_STACK: 16777216
```

Comment:
You probably don't need the min stack, but it won't hurt either. We have this in rayon because some of its tests were recursing too deeply in debug mode, without inlining etc. (I think that was actually on Linux, but we choose to apply the min stack everywhere.)

Reply:
I figured it probably wasn't necessary, but was just being lazy :-) I'll take it out for now...